### PR TITLE
fix selector content fixes #652

### LINKED ISSUE #652: [facebook bug] art camps and theme camps shouldn't appear in the same list
לא הגיוני כל כך שמחנות ואומנות מופיעים באותה רשימה. זה גם יוצר בילבול וגם לא יאפשר לאנשים להרשם גם וגם. והרי הרבה אומנים הם גם חברי מחנה

### PR DESCRIPTION
**Pull request by Aroundafur**
### Proposed solution
in selector, filter camps for views.
select camp that are and aren't art_installations

### Tradeoffs
none

### Testing Done
go to db and find an art_installation camp according to midburn year.
this item should show for the selector in /art-admin and not in /camps

### CODE REVIEW

**Comment by Aroundafur** [comment]
@NitayRabi 
@amotenko

File: views/pages/camps/index_admin.jade [hunk 1]
```diff
@@ -56,7 +56,7 @@ block content
                             //- th.admin-edit=t(t_prefix+'stats.edit')
                             //- th.admin-remove=t(t_prefix+'stats.remove')
                     tbody
-                        tr(ng-repeat="camp in camps | filter: searchCamp | orderBy: orderCamps")
```
Comment (amotenko) [inline views/pages/camps/index_admin.jade:59]: As discussed, this breaks textual search functionality

File: views/pages/camps/index_user.jade [hunk 2]
```diff
@@ -17,12 +17,12 @@ block content
                                     label=t('camps:new.name_en')
                                     input.hidden(id='join_camp_request_join_user_id', type='hidden', value='#{user.attributes.user_id}')
                                     select(type="text", id='join_camp_request_camp_id', class="form-control", name="camp_name_en", autofocus="true", required)
-                                        option(ng-repeat="camp in camps", value="{{camp.id}}") {{camp.camp_name_en}}
+                                        option(ng-repeat="camp in camps | filter:{__prototype: '!art_installation'}", value="{{camp.id}}") {{camp.camp_name_en}}
```
Comment (amotenko) [inline views/pages/camps/index_user.jade:20]: Better to be explicit on the kinds of values you expect here, and not implicit.
Don't forget there may be other types  as well beside those two

**Review by amotenko — CHANGES_REQUESTED** [review]

File: views/pages/camps/index_admin.jade (hunk 1 above)
Comment (Aroundafur) [reply views/pages/camps/index_admin.jade:59]: fixed

File: views/pages/camps/index_user.jade (hunk 2 above)
Comment (Aroundafur) [reply views/pages/camps/index_user.jade:20]: thought you asked to only remove art_installation camps from this dropdown.

If it's more complex, we should consider a different approach.

Comment (NitayRabi) [reply views/pages/camps/index_user.jade:20]: @Aroundafur - Try searching for an option of passing an array of values to the filter param and pass the correct values (you can store it in a var in angularJS side)

**Pull request closed without merging**